### ISSUE TITLE
Move Repository To rust-lang

### ISSUE THREAD

**Issue by XAMPPRocky** [rust-lang-nursery/rust-cookbook#558]
Hello, the current `rust-lang-nursery` organisation is considered deprecated, and the Rust Programming Language organisation is trying to consolidate managing Rust's GitHub organisations, as such we'd like you to consider moving your repository to the main `rust-lang` organisation instead. 

### How Do I Move To `rust-lang`?
You won't be able to transfer your repository directly, you'll need to transfer your repository to @Mark-Simulacrum, who will then move your repository to the `rust-lang` organisation.

### Deprecating
If you no longer intend to maintain this repository, please let us know and we will take of deprecating the project.

**Comment by AndyGauge** [comment]
I know Mark, but I'm not familiar with this process change.  I don't think it is wise to transfer the repository without a more official communication.  Can you link the announcement that rust-lang-nursery is deprecated?

**Comment by XAMPPRocky** [comment]
@AndyGauge Hello, unfortunately there is no annoucement post for the deprecation. You can read where the Governance WG discussed this topic with core team members in the following discord link or by searching `in: wg-governance rust-lang-nursery` in the Rust Programming Language. This was a decision that was made awhile ago by the core team, but the work was never finished. This issue acts a continuation of that.

https://discordapp.com/channels/442252698964721669/557273736118796289/634122262143959041

**Comment by AndyGauge** [comment]
Yeah, that's official enough for me.  It was kinda out of the blue and I couldn't find anything on rust users about it.

**Comment by AndyGauge** [comment]
Whoops looks like that's above my pay grade here.  I don't have the settings on the main page, just write permissions.

**Comment by AndyGauge** [comment]
So I just realized that our hosting is tied to nursery.  All of the google results, all of the external links are pointing to https://rust-lang-nursery.github.io/rust-cookbook/
While I would be fine moving the repo, I'd really want to make sure the hosting continues to resolve.  That's a bit of a challenge, and while I know github aliases renamed repositories, I have no idea what it does for jekyll based deployments.

**Comment by XAMPPRocky** [comment]
@AndyGauge I believe @pietroalbini has already set up a redirect so that any `rust-lang-nursery.github.io` 404 redirects to `rust-lang.github.io`.

**Comment by AndyGauge** [comment]
Tried email to get in touch with @Mark-Simulacrum to move over.  2024 revamp at minimum.
